doc/whats_new/v0.20.rst

```diff
@@ -215,6 +215,11 @@ Classifiers and regressors
   unexpected TypeError given dense inputs.
   :issue:`10412` by :user:`Jiongyan Zhang <qmick>`.
 
+- Fixed a bug in :class:`tree._splitter.BestSplitter` and
```

Comment:
Please refer to it by public API elements, i.e. without reference to `_splitter`

Reply:
Should I refer to `tree.DecisionTreeClassifier` and `tree.DecisionTreeRegressor` ?

Reply:
You can just say decision trees, which covers forests, etc. Mention the splitter parameter